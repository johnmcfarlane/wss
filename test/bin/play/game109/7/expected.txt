 30 11,6  | GOATEE
 30 11,6  | GOETIA
 25  1,7  | AOTEA
 24 11,2  | EATAGE
 24 11,4  | EOGAEA
 21 11,5  | AGATE
 21 11,5  | AGETE
 21  9,4  | AILE
 21  9,4  | AOLI
 21 11,6  | GETAE
 21 11,3  | TAIGA
 20  1,8  | IOTA
 20  1,8  | ITEA
 19  9,4  | AIL
 19  7,1  | ITEA
 18 11,6  | GAEA
 18 11,6  | GAET
 18 11,6  | GAIA
 18 11,6  | GAIT
 18 11,6  | GATA
 18 11,6  | GATE
 18 11,6  | GEAT
 18 11,6  | GEET
 18 11,6  | GETA
 18 11,6  | GITE
 18 11,6  | GOAT
 18 11,6  | GOTE
 18 11,3  | TAIG
 18  7,2  | TEA
 18  7,2  | TEE
 18 11,3  | TOAG
 17  3,8  | IOTA
 17  3,8  | ITEA
 17  1,6  | TOEA
 15  5,6  | AOTEA
 14  9,7  - ATI
 14  7,3  | EE
 14  6,4  - SEZ
 13  1,7  | AET
 13  1,7  | AIT
 13  1,7  | EAT
 13  1,7  | OAT
 13  9,5  | OLEATE
 13  8,4  - ZATI
 13  8,4  - ZETA
 13  8,4  - ZOEA
 12  5,6  | AET
 12  5,6  | AIT
 12  5,6  | ATE
 12  5,6  | ATI
 12  5,6  | EAT
 12  9,5  | ELATE
 12  9,5  | ELIOT
 12  9,5  | ELITE
 12  5,6  | ETA
 12  5,6  | OAT
 12  8,4  - ZAT
 12  8,4  - ZEE
 12  8,4  - ZIT
 12  8,4  - ZOA
 11  5,6  | AA
 11  5,6  | AE
 11  1,8  | AET
 11  5,6  | AI
 11  1,8  | AIT
 11  5,6  | AT
 11  1,8  | ATE
 11  1,8  | ATI
 11  1,8  | EAT
 11  5,6  | EE
 11  5,6  | EO
 11  5,6  | ET
 11  1,8  | ETA
 11  1,8  | ITA
 11  1,8  | OAT
 11  5,6  | OE
 11  5,6  | OI
 11  8,4  - ZA
 10 10,2  | AOTEA
 10 10,6  | AOTEA
 10  9,5  | ELIA
 10  9,5  | ILEA
 10  9,5  | ILOT
 10  8,5  - OAT
 10  9,5  | OLEA
  9  9,7  - AA
  9  9,7  - AE
  9  9,7  - AI
  9  9,7  - AT
  9  5,6  - AX
  9  9,5  | ELT
  9  5,6  - EX
  9  9,5  | ILA
  9  9,5  | ILE
  9  9,7  - IT
  9  9,7  - OE
  9  9,7  - OI
  9  9,5  | OLA
  9  9,5  | OLE
  9  5,6  - OX
  8 10,5  - AA
  8  1,8  | AA
  8  1,8  | AE
  8 10,7  - AI
  8  1,8  | AI
  8  1,8  | AT
  8  1,8  | EE
  8  3,7  | EO
  8 10,7  - EO
  8  1,8  | EO
  8  1,8  | ET
  8 10,7  - IO
  8  1,8  | IO
  8  1,8  | IT
  8  9,6  | LAETI
  8  1,8  | OE
  8  1,8  | OI
  8 10,5  - TA
  8  1,6  | TAA
  8  1,6  | TAE
  8  1,6  | TAI
  8  1,6  | TAO
  8  1,6  | TEA
  8  1,6  | TEE
  8 10,7  - TI
  8 10,7  - TO
  8  1,6  | TOA
  8  1,6  | TOE
  8  1,6  | TOI
  7  3,6  | EE
  7  9,5  | EL
  7  9,6  | LAET
  7  9,6  | LAIT
  7  9,6  | LATE
  7  9,6  | LATI
  7  9,6  | LEAT
  7  9,6  | LEET
  7  9,6  | LITE
  7  9,6  | LOTA
  7  8,5  - OE
  7  3,6  | OE
  7  1,8  - OSTIA
  7  2,8  - SAETA
  7  2,8  - SAITE
  7  2,8  - SATAI
  7  2,8  - SETAE
  7  2,8  - SITAO
  7  2,8  - SOTIE
  7  2,8  - STAIA
  7  2,8  - STAIO
  7  2,8  - STOAE
  7  2,8  - STOAI
  7  3,6  | TE
  7 11,4  | TOGAE
  6  1,7  | AA
  6  1,7  | AE
  6 11,5  | AGEE
  6 11,5  | AGET
  6 11,5  | AGIO
  6 11,5  | AGIT
  6  1,7  | AI
  6  1,7  - ARATI
  6  1,7  - AREAE
  6  1,7  - ARETE
  6  1,7  - ARIOT
  6  1,7  - ARITE
  6  1,8  - ASEA
  6  1,8  - ASIA
  6  1,8  - ASTA
  6  7,1  | ATE
  6  1,7  | EE
  6  1,7  | EO
  6  1,7  - ERATO
  6 11,4  | IAGO
  6  1,7  - IRATE
  6  3,8  | ITA
  6  3,8  | OAT
  6  1,7  | OE
  6 11,5  | OGEE
  6  1,7  | OI
  6  1,7  - ORATE
  6  2,8  - SATA
  6  2,8  - SATE
  6  2,8  - SATI
  6  2,8  - SEAT
  6  2,8  - SEIT
  6  2,8  - SETA
  6  2,8  - SITA
  6  2,8  - SITE
  6  2,8  - STOA
  6 11,4  | TEGA
  6 11,4  | TIGE
  6 11,4  | TOGA
  6  1,8  - TSIA
  5  1,6  | AA
  5  1,6  | AE
  5 11,5  | AGA
  5 11,5  | AGE
  5 11,5  | AGO
  5 11,5  | AGT
  5  1,7  - AREA
  5  1,7  - ARET
  5  1,7  - ARIA
  5  1,7  - ARTE
  5  1,6  | EE
  5 11,5  | EGO
  5  1,6  | EO
  5  1,7  - ERAT
  5  1,7  - ERIA
  5  1,7  - ERIE
  5 11,6  | GAE
  5 11,6  | GAT
  5 11,6  | GEE
  5 11,6  | GEO
  5 11,6  | GET
  5 11,6  | GIE
  5 11,6  | GIO
  5 11,6  | GIT
  5 11,6  | GOA
  5 11,6  | GOI
  5 11,6  | GOT
  5 11,6  | GTE
  5  3,8  | IO
  5  3,8  | IT
  5  9,6  | LAI
  5  9,6  | LAO
  5  9,6  | LAT
  5  9,6  | LEA
  5  9,6  | LEE
  5  9,6  | LEI
  5  9,6  | LET
  5  9,6  | LIE
  5  9,6  | LIT
  5  9,6  | LOA
  5  9,6  | LOE
  5  9,6  | LOT
  5  1,6  | OE
  5  3,8  | OE
  5  3,8  | OI
  5  1,7  - ORAE
  5  2,7  - RAIA
  5  2,7  - RATA
  5  2,7  - RATE
  5  2,7  - RATO
  5  2,7  - REET
  5  2,7  - REIT
  5  2,7  - RETE
  5  2,7  - RIOT
  5  2,7  - RITA
  5  2,7  - RITE
  5  2,7  - ROIT
  5  2,7  - ROTA
  5  2,7  - ROTE
  5  2,7  - ROTI
  5  2,8  - SAA
  5  2,8  - SAE
  5  2,8  - SAI
  5  2,8  - SAO
  5  2,8  - SAT
  5  2,8  - SEA
  5  2,8  - SEE
  5  2,8  - SEI
  5  2,8  - SET
  5  2,8  - SIA
  5  2,8  - SIE
  5  2,8  - SIT
  5  2,8  - SOE
  5  2,8  - SOT
  5  2,8  - STA
  5  1,6  | TA
  5 11,4  | TAG
  5  7,2  | TE
  5  1,6  | TE
  5 11,4  | TEG
  5 11,4  | TIG
  5  1,6  | TO
  5 11,4  | TOG
  5  1,7  - TREE
  5  1,7  - TRIO
  5  6,2  - UT
  4 11,5  | AG
  4  1,7  - ARE
  4  1,7  - ARO
  4  1,7  - ART
  4  1,7  - ERA
  4  1,7  - ERE
  4 11,6  | GI
  4 11,6  | GO
  4 10,3  | IOTA
  4  1,7  - IRA
  4  1,7  - IRE
  4 10,3  | ITEA
  4  9,6  | LA
  4  9,6  | LI
  4  9,6  | LO
  4  1,7  - ORA
  4  1,7  - ORE
  4  1,7  - ORT
  4  2,7  - RAT
  4  2,7  - REA
  4  2,7  - REE
  4  2,7  - RET
  4  2,7  - RIA
  4  2,7  - RIE
  4  2,7  - RIO
  4  2,7  - RIT
  4  2,7  - ROE
  4  2,7  - ROI
  4  2,7  - ROT
  4  2,7  - RTE
  4  2,7  - RTI
  4 10,4  | TIAO
  4 10,3  | TOEA
  4  1,7  - TRA
  4  1,7  - TRI
  3 10,4  | AAA
  3 10,5  | AAA
  3 10,6  | AAA
  3 10,6  | AET
  3 10,6  | AIT
  3 10,6  | ATE
  3 10,6  | ATI
  3 10,5  | EAT
  3  1,8  - ESE
  3  1,8  - EST
  3 10,4  | ETA
  3  1,8  - ISE
  3  1,8  - IST
  3 10,4  | ITA
  3 10,5  | OAT
  3  1,8  - OSE
  3  2,7  - RE
  3 10,4  | TAA
  3 10,5  | TAA
  3 10,5  | TAE
  3 10,5  | TAI
  3 10,5  | TAO
  3 10,4  | TEA
  3 10,4  | TOA
  2 10,5  | AA
  2 10,6  | AA
  2 10,6  | AE
  2 10,6  | AI
  2  1,7  - AR
  2  1,8  - AS
  2 10,6  | AT
  2  1,6  - AeT
  2  1,7  - ER
  2  1,8  - ES
  2  1,8  - IS
  2  1,7  - OR
  2  1,8  - OS
  2  2,8  - SI
  2  2,8  - SO
  2 10,5  | TA
  2  1,6  - TeA
  2  1,6  - TeE
  1  1,6  - Ae
  1  1,6  - Ee
  1  1,6  - Oe
  1  1,6  - Te
  1  2,6  - eE
  1  2,6  - eO
  1  2,6  - eT
