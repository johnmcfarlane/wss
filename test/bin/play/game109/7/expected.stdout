 30 11,6  | GOATEE
 21 11,5  | AGATE
 21 11,5  | AGITA
 21 11,3  | TAIGA
 20  1,8  | IOTA
 19  9,4  | AIL
 18 11,6  | GAIT
 18 11,6  | GATE
 18 11,6  | GETA
 18 11,6  | GITE
 18 11,6  | GOAT
 17  3,8  | IOTA
 17  1,6  | TOEA
 13  1,7  | AIT
 13  1,7  | EAT
 13  1,7  | OAT
 13  8,4  - ZETA
 13  8,4  - ZOEA
 12  5,6  | AIT
 12  5,6  | ATE
 12  5,6  | EAT
 12  9,5  | ELATE
 12  9,5  | ELITE
 12  5,6  | ETA
 12  5,6  | OAT
 12  8,4  - ZEE
 12  8,4  - ZIT
 12  8,4  - ZOA
 11  5,6  | AA
 11  5,6  | AE
 11  5,6  | AI
 11  1,8  | AIT
 11  5,6  | AT
 11  1,8  | ATE
 11  1,8  | EAT
 11  5,6  | EE
 11  5,6  | ET
 11  1,8  | ETA
 11  1,8  | OAT
 11  5,6  | OE
 11  5,6  | OI
 11  8,4  - ZA
 10  9,5  | ILEA
 10  8,5  - OAT
  9  9,7  - AA
  9  9,7  - AE
  9  9,7  - AI
  9  9,7  - AT
  9  5,6  - AX
  9  5,6  - EX
  9  9,7  - IT
  9  9,7  - OE
  9  9,7  - OI
  9  5,6  - OX
  8 10,5  - AA
  8  1,8  | AA
  8  1,8  | AE
  8 10,7  - AI
  8  1,8  | AI
  8  1,8  | AT
  8  1,8  | EE
  8  1,8  | ET
  8  1,8  | IT
  8  1,8  | OE
  8  1,8  | OI
  8 10,5  - TA
  8  1,6  | TAE
  8  1,6  | TAO
  8  1,6  | TEA
  8  1,6  | TEE
  8 10,7  - TI
  8 10,7  - TO
  8  1,6  | TOE
  7  3,6  | EE
  7  9,5  | EL
  7  9,6  | LATE
  7  9,6  | LATI
  7  9,6  | LEET
  7  9,6  | LITE
  7  9,6  | LOTA
  7  9,6  | LOTI
  7  8,5  - OE
  7  1,8  - OSTIA
  7  2,8  - SETAE
  7  2,8  - STOAE
  7  2,8  - STOAI
  7  3,6  | TE
  7 11,4  | TOGAE
  6  1,7  | AA
  6  1,7  | AE
  6 11,5  | AGEE
  6 11,5  | AGIO
  6  1,7  | AI
  6  1,7  - AREAE
  6  1,7  - ARETE
  6  1,8  - ASEA
  6  7,1  | ATE
  6  1,7  | EE
  6  1,7  - IRATE
  6  3,8  | OAT
  6  1,7  | OE
  6 11,5  | OGEE
  6  1,7  | OI
  6  1,7  - ORATE
  6  2,8  - SATE
  6  2,8  - SATI
  6  2,8  - SEAT
  6  2,8  - SETA
  6  2,8  - SITE
  6  2,8  - STOA
  6 11,4  | TOGA
  5  1,6  | AA
  5  1,6  | AE
  5 11,5  | AGA
  5 11,5  | AGE
  5 11,5  | AGO
  5  1,7  - AREA
  5  1,7  - ARIA
  5  1,6  | EE
  5 11,5  | EGO
  5 11,6  | GAE
  5 11,6  | GAT
  5 11,6  | GEE
  5 11,6  | GET
  5 11,6  | GIE
  5 11,6  | GIT
  5 11,6  | GOA
  5 11,6  | GOT
  5  3,8  | IT
  5  9,6  | LAT
  5  9,6  | LEA
  5  9,6  | LEE
  5  9,6  | LEI
  5  9,6  | LET
  5  9,6  | LIE
  5  9,6  | LIT
  5  9,6  | LOT
  5  1,6  | OE
  5  3,8  | OE
  5  3,8  | OI
  5  2,7  - RAIA
  5  2,7  - RATE
  5  2,7  - RATO
  5  2,7  - RETE
  5  2,7  - RIOT
  5  2,7  - RITE
  5  2,7  - ROTA
  5  2,7  - ROTE
  5  2,7  - ROTI
  5  2,8  - SAE
  5  2,8  - SAT
  5  2,8  - SEA
  5  2,8  - SEE
  5  2,8  - SEI
  5  2,8  - SET
  5  2,8  - SIT
  5  2,8  - SOT
  5  1,6  | TA
  5 11,4  | TAG
  5  7,2  | TE
  5  1,6  | TE
  5 11,4  | TEG
  5  1,6  | TO
  5 11,4  | TOG
  5  1,7  - TREE
  5  1,7  - TRIO
  5  6,2  - UT
  4 11,5  | AG
  4  1,7  - ARE
  4  1,7  - ART
  4  1,7  - ERA
  4  1,7  - ERE
  4 11,6  | GI
  4 11,6  | GO
  4 10,3  | IOTA
  4  1,7  - IRE
  4  9,6  | LA
  4  9,6  | LI
  4  9,6  | LO
  4  1,7  - ORA
  4  1,7  - ORE
  4  1,7  - ORT
  4  2,7  - RAI
  4  2,7  - RAT
  4  2,7  - REE
  4  2,7  - RET
  4  2,7  - RIA
  4  2,7  - ROE
  4  2,7  - ROT
  4 10,3  | TOEA
  3 10,6  | AIT
  3 10,6  | ATE
  3 10,5  | EAT
  3 10,4  | ETA
  3 10,5  | OAT
  3  1,8  - OSE
  3  2,7  - RE
  3 10,5  | TAE
  3 10,5  | TAO
  3 10,4  | TEA
  2 10,5  | AA
  2 10,6  | AA
  2 10,6  | AE
  2 10,6  | AI
  2  1,7  - AR
  2  1,8  - AS
  2 10,6  | AT
  2  1,7  - ER
  2  1,8  - ES
  2  1,8  - IS
  2  1,7  - OR
  2  1,8  - OS
  2  2,8  - SI
  2  2,8  - SO
  2 10,5  | TA
  2  1,6  - TeA
  2  1,6  - TeE
  1  1,6  - Ae
  1  1,6  - Ee
  1  1,6  - Oe
  1  1,6  - Te
  1  2,6  - eE
  1  2,6  - eT
